src/Catty/Parser&Serialializer/Old Parser/ProjectParser.m

```diff
@@ -621,6 +621,10 @@ - (NSString*)classNameForString:(NSString*)classString
     else if([className isEqualToString:@"ChangeGhostEffectByNBrick"]) {
         className = @"ChangeTransparencyByNBrick";
     }
+    
+    else if([className isEqualToString:@"PenBrick"]) {
```

Comment:
This can be removed from the old parser since this Brick was not available at this CatrobatLanguageVersion.